android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "29.0.2"
-        minSdkVersion = 16
+        minSdkVersion = 21
```

Comment:
This appears to be a requirement in order to use the Plaid SDK on Android

https://github.com/plaid/react-native-plaid-link-sdk#3-configure-gradle

I'm hoping it's OK ? @Jag96

Reply:
Oh nice!